### PR TITLE
Added geetest support for deathbycaptcha.com service

### PR DESCRIPTION
**Pull request by david96182**
Recently deathbycaptcha added support for **geetest** v3 and v4. I add support for this type of captchas in unicaps and also update the tests to include them for deathbycaptcha.
On the other hand, update the tables of the characteristics of the captchas that deathbycaptcha supports, there are some that are marked as not supported when it does. Although their documentation doesn't say so explicitly, they do support it. You can try. These are:

1. Image Case Sensitive
2. Image Numbers only
3. Image Letters only
4. Image Math
5. reCaptcha v2 Enterprise
6. reCaptcha v2 Google service
7. reCaptcha v3 Enterprise

### CODE REVIEW

File: unicaps/_service/deathbycaptcha.py [hunk 1]
```diff
@@ -234,7 +236,16 @@ def parse_response(self, response) -> dict:
 
         # get solution class and prepare a solution object
         solution_class = self.source_data['task'].captcha.get_solution_class()
-        solution = solution_class(text)
+        captcha_type = self.source_data['task'].captcha.get_type()
+        args = []
+        kwargs = {}
+        print(text)
+        if captcha_type in (CaptchaType.GEETEST, CaptchaType.GEETESTV4):
+            kwargs.update(eval(text))
```
Comment (sergey-scat) [inline unicaps/_service/deathbycaptcha.py]: Plz do not use `eval()`
https://stackoverflow.com/questions/1832940/why-is-using-eval-a-bad-practice

File: unicaps/_service/deathbycaptcha.py [hunk 2]
```diff
@@ -234,7 +236,16 @@ def parse_response(self, response) -> dict:
 
         # get solution class and prepare a solution object
         solution_class = self.source_data['task'].captcha.get_solution_class()
-        solution = solution_class(text)
+        captcha_type = self.source_data['task'].captcha.get_type()
+        args = []
+        kwargs = {}
+        print(text)
```
Comment (sergey-scat) [inline unicaps/_service/deathbycaptcha.py]: Remove this

File: README.md [hunk 3]
```diff
@@ -83,7 +83,7 @@ if __name__ == '__main__':
 | [azcaptcha.com](https://azcaptcha.com/) | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | Latin | ✅ |
 | [captcha.guru](https://captcha.guru/ru/reg/?ref=127872) | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | Latin | ✅ |
 | [cptch.net](https://cptch.net/auth/signup?frm=0ebc1ab34eb04f67ac320f020a8f709f) | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | Cyrillic/Latin | ❌ |
-| [deathbycaptcha.com](http://deathbycaptcha.com) | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ | Latin | ❌ |
+| [deathbycaptcha.com](http://deathbycaptcha.com) | ✅ | ✅ | ❌ | ✅ | ✅ | ✅ | ❌ | Latin | ❌ |
```
Comment (sergey-scat) [inline README.md:86]: 1. deathbycaptcha does not provide an option to specify the flags "Case Sensitive", "Numbers only", etc.
2. What does this have to do with "Added geetest support for deathbycaptcha.com service"?

Please revert this changes.

File: README.md [hunk 4]
```diff
@@ -112,7 +112,7 @@ if __name__ == '__main__':
 | [azcaptcha.com](https://azcaptcha.com/) | ✅ | ✅ | ❌ | ✅ | ✅ | ✅ | ✅ |
 | [captcha.guru](https://captcha.guru/ru/reg/?ref=127872) | ✅ | ✅ | ❌ | ✅ | ✅ | ✅ | ✅ |
 | [cptch.net](https://cptch.net/auth/signup?frm=0ebc1ab34eb04f67ac320f020a8f709f) | ✅ | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ |
-| [deathbycaptcha.com](http://deathbycaptcha.com) | ✅ | ✅ | ❌ | ❌ | ✅ | ❌ | ❌ |
+| [deathbycaptcha.com](http://deathbycaptcha.com) | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ | ❌ |
```
Comment (sergey-scat) [inline README.md:115]: Agreed with the "Google service" support only, the Enterprise reCAPTCHA v2 is not supported

File: README.md [hunk 5]
```diff
@@ -128,7 +128,7 @@ if __name__ == '__main__':
 | [azcaptcha.com](https://azcaptcha.com/) | ✅ | ❌ | ✅ | ❌ | ❌ |
 | [captcha.guru](https://captcha.guru/ru/reg/?ref=127872) | ✅ | ❌ | ✅ | ✅ | ✅ |
 | [cptch.net](https://cptch.net/auth/signup?frm=0ebc1ab34eb04f67ac320f020a8f709f) | ✅ | ❌ | ❌ | ❌ | ❌ |
-| [deathbycaptcha.com](http://deathbycaptcha.com) | ✅ | ❌ | ✅ | ❌ | ❌ |
+| [deathbycaptcha.com](http://deathbycaptcha.com) | ✅ | ✅ | ✅ | ❌ | ❌ |
```
Comment (sergey-scat) [inline README.md:131]: Enterprise reCAPTCHA v3 is not supported

**Review by sergey-scat — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (david96182) [reply README.md:115]: I have solved this type of captchas, it does not have a very good percentage of success, but it does solve it.

File: README.md (hunk 5 above)
Comment (david96182) [reply README.md:131]: I have solved this type of captchas, it does not have a very good percentage of success, but it does solve it.

File: README.md (hunk 3 above)
Comment (david96182) [reply README.md:86]: 1. They do not have implemented the parameters to specify the details of the captcha of type image, but they do solve all the ones I mention.
2. Sorry. Since it is related to deathbycaptcha.com I included it here.

File: unicaps/_service/deathbycaptcha.py (hunk 1 above)
Comment (david96182) [reply unicaps/_service/deathbycaptcha.py]: Done. Changed by json.loads replacing the single quotes.

File: README.md (hunk 5 above)
Comment (sergey-scat) [reply README.md:131]: Give me an example please

File: README.md (hunk 4 above)
Comment (sergey-scat) [reply README.md:115]: Give me an example please

File: README.md (hunk 3 above)
Comment (sergey-scat) [reply README.md:86]: Once again, deathbycaptcha does not allow to pass the values of these parameters.

File: README.md (hunk 4 above)
Comment (david96182) [reply README.md:115]: I tried with your example recaptcha_v2_enterprise.py and it returns the captcha response but it doesn't solve it. I tried with Twocaptcha and it does not solve it either, it returns the answer but it does not solve it. In fact, deathbycaptcha gets the response much faster.

File: README.md (hunk 3 above)
Comment (david96182) [reply README.md:86]: It is true that it does not allow to pass the parameters, but it does solve those types of images. I think you should clarify the table to which it corresponds, because at least I understand the types of captcha of image type that each service resolves and not the parameters that it accepts, for that I think that the documentation of each service is already there.

File: README.md (hunk 5 above)
Comment (david96182) [reply README.md:131]: Try https://www.netflix.com/login
